ghcide/src/Development/IDE/Core/Compile.hs

```diff
@@ -989,12 +990,20 @@ mkDetailsFromIface session iface linkable = do
     initIfaceLoad hsc' (typecheckIface iface)
   return (HomeModInfo iface details linkable)
 
+fakeSpan :: RealSrcSpan
+fakeSpan = realSrcLocSpan $ mkRealSrcLoc (Util.fsLit "<ghcide>") 1 1
```

Comment:
Moving this into the top-down ordering of the process of abstraction creation, it is used below, so moving it here.